fix: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password

liyigang1 opened the pull request:
When checking file information, if the host is down and can be accessed, a prompt box will pop up

Log: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password
Bug: https://pms.uniontech.com/bug-view-261605.html

liyigang1 committed:
fix: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password

When checking file information, if the host is down and can be accessed, a prompt box will pop up

Log: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password
Bug: https://pms.uniontech.com/bug-view-261605.html

Johnson-zs commented on src/plugins/filemanager/core/dfmplugin-workspace/views/fileview.cpp:
```diff
@@ -1915,6 +1917,18 @@ void FileView::initializeConnect()
     }
+    connect(&FileInfoHelper::instance(), &FileInfoHelper::smbSeverMayModifyPassword, this, [this](const QUrl &url){
+        if (DeviceUtils::isSamba(rootUrl()) && url.path().startsWith(rootUrl().path())) {
+            qInfo() << rootUrl() << url << "smb server may modify password";
```
fmInfo

liyigang1 replied:
已修改

Johnson-zs commented on src/dfm-base/file/local/asyncfileinfo.h:
```diff
@@ -65,6 +65,7 @@ class AsyncFileInfo : public FileInfo
     // less 0,cache fialed, equeal 0,another cache, bigger 0 cache success
     int cacheAsyncAttributes();
     bool asyncQueryDfmFileInfo(int ioPriority = 0, initQuerierAsyncCallback func = nullptr, void *userData = nullptr);
+    DFMIOError errorDfm();
```
这个函数作为 const 函数好一些，其次名字很奇怪，叫 errorFromDFMIO ？

liyigang1 replied:
已修改

liyigang1 committed:
fix: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password

update translation

Log: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password
Bug: https://pms.uniontech.com/bug-view-261605.html

liyigang1 commented:
/merge
